tests/query/test_timeseries_extension.py

```diff
@@ -23,6 +48,7 @@
             ("timestamp", ">=", "2019-09-19T10:00:00"),
             ("timestamp", "<", "2019-09-19T12:00:00"),
         ],
+        build_time_condition(datetime(2019, 9, 19, 10), datetime(2019, 9, 19, 12)),
```

Comment:
Similar comment as `build_in` previously — i think `build_time_condition("timestamp", datetime(2019, 9, 19, 10), datetime(2019, 9, 19, 12))` would be a bit clearer.